R/reportEDGETransport.R

```diff
@@ -520,7 +527,8 @@ reportEDGETransport <- function(output_folder=".",
     toMIF <- rbindlist(list(
       toMIF,
       toMIF[region %in% regionSubsetList[[1]],.(value = sum(value), region = "EUR"), by = .(model, scenario, variable, unit, period)],
-      toMIF[region %in% regionSubsetList[[2]],.(value = sum(value), region = "NEU"), by = .(model, scenario, variable, unit, period)]
+      toMIF[region %in% regionSubsetList[[2]],.(value = sum(value), region = "NEU"), by = .(model, scenario, variable, unit, period)],
```

Comment:
are you sure this should be linked to the *same* regionSubsetList? So is `NEU == EU27`?